### ISSUE TITLE
Can not open file for read - although the file exists.

### ISSUE THREAD

**Issue by IngwiePhoenix** [graphitemaster/incbin#19]
So all I did was:

```
$ gcc incbin/incbin.c -o incbin
$ ./incbin -o scripts.cpp src/scripts.rc
```

and got an error:

```
Ingwie@Ingwies-Macbook-Pro.local ~/W/IceTea $ file src/scripts.rc 
src/scripts.rc: ASCII c program text
Ingwie@Ingwies-Macbook-Pro.local ~/W/IceTea $ ./out/incbin -o scripts.cpp ./src/scripts.rc 
failed to open `./src/scripts.rc' for reading
```

What exactly could be the issue for this?

**Comment by IngwiePhoenix** [comment]
In fact, this is the whole script: https://github.com/IngwiePhoenix/IceTea/blob/master/build.sh (will be commenting out the line that calls incbin soon, its actually not even required on UNIX :). But its analogous to the windows variant.)

**Comment by graphitemaster** [comment]
Fixed in 76a0059
